### PR TITLE
Some change

### PR DESCRIPTION
**Pull request by ging-dev**
- Translate comments
- Friendly URL for guestbook

### CODE REVIEW

File: config/routes.php [hunk 1]
```diff
@@ -22,13 +22,13 @@
     $map->addRoute(['GET', 'POST'], '/album[/]', 'modules/album/index.php');                         // Photo Album
     $map->addRoute(['GET', 'POST'], '/downloads[/]', 'modules/downloads/index.php');                 // Downloads
     $map->addRoute(['GET', 'POST'], '/forum[/]', 'modules/forum/index.php');                         // Forum
-    $map->addRoute(['GET', 'POST'], '/guestbook[/]', 'modules/guestbook/index.php');                 // Guestbook, mini-chat
+    $map->addRoute(['GET', 'POST'], '/guestbook[/[{action}[/[{id:\d+}[/]]]]]', 'modules/guestbook/index.php');                 // Guestbook, mini-chat
```
Comment (batumibiz) [inline config/routes.php]: Please don't make unnecessary changes to one commit. If commit is dedicated to translating comments into English, that's all it should be. Changing Route makes a problem to Guestbook.

File: config/routes.php [hunk 2]
```diff
@@ -22,13 +22,13 @@
...
-    $map->addRoute(['GET', 'POST'], '/guestbook[/[{action}[/[{id:\d+}[/]]]]]', 'modules/guestbook/index.php');                 // Guestbook, mini-chat
+    $map->addRoute(['GET', 'POST'], '/guestbook[/[{action}]]', 'modules/guestbook/index.php');                 // Guestbook, mini-chat
     $map->addRoute(['GET', 'POST'], '/help[/]', 'modules/help/index.php');                           // Help
     $map->addRoute(['GET', 'POST'], '/library[/]', 'modules/library/index.php');                     // Articles Library
     $map->addRoute(['GET', 'POST'], '/language[/]', 'modules/language/index.php');                   // Language switcher
     $map->addRoute(['GET', 'POST'], '/login[/]', 'modules/login/index.php');                         // Login / Logout
     $map->addRoute(['GET', 'POST'], '/mail[/]', 'modules/mail/index.php');                           // Personal Messages
-    $map->addRoute(['GET', 'POST'], '/news[/[{action}[/[{id:\d+}[/]]]]]', 'modules/news/index.php'); // News
+    $map->addRoute(['GET', 'POST'], '/news[/[{action}]]', 'modules/news/index.php'); // News
```
Comment (batumibiz) [inline config/routes.php]: What's this for???
This change breaks news.

Comment (ging-dev) [reply config/routes.php]: oh, I have changed the links in the `news/templates` and it will operate normally with url `domain/modules/act?id=`

File: config/routes.php (hunk 1 above)
Comment (ging-dev) [reply config/routes.php]: I will pay attention to it in the next time

File: config/routes.php (hunk 2 above)
Comment (simba77) [reply config/routes.php]: What is the reason for these changes?

File: config/routes.php (hunk 1 above)
Comment (ging-dev) [reply config/routes.php]: I have consulted some people to make this decision. The change also makes the router simpler and more concise later 😆

File: config/routes.php (hunk 2 above)
Comment (ging-dev) [reply config/routes.php]: I have consulted some people to make this decision. The change also makes the router simpler and more concise later 😊

Comment (simba77) [reply config/routes.php]: For simplicity you can make route rule as: `$map->addRoute(['GET', 'POST'], '/news/[{action}/[{id:\d+}]]', 'modules/news/index.php');` and replace urls in templates for this rule.

Comment (ging-dev) [reply config/routes.php]: In my opinion, the `id` parameter does not represent a file. Therefore it is not so important on URL. If you still want that, I will do it.

Comment (simba77) [reply config/routes.php]: OK. In our case, this is not critical, but in general, if you make a friendly URL, it is better to do it completely.
